src/Error/ExceptionRenderer.php

```diff
@@ -276,7 +276,20 @@ public function render(): ResponseInterface
         }
         $this->controller->setResponse($response);
 
-        return $this->_outputMessage($template);
+        try {
+            return $this->renderTemplate($template);
+        } catch (MissingTemplateException $e) {
+            if (!$e instanceof MissingLayoutException) {
```

Comment:
This is due to MissingLayoutException extending MissingTemplateException. I could either catch and re-throw it like in renderTemplate() or do this. This seemed cleaner for this one scenario.